packages/teuchos/comm/src/Teuchos_StackedTimer.hpp

```diff
@@ -589,13 +589,14 @@ class StackedTimer
    */
   struct OutputOptions {
     OutputOptions() : output_fraction(false), output_total_updates(false), output_histogram(false),
-                      output_minmax(false), num_histogram(10), max_levels(INT_MAX),
+                      output_minmax(false), output_proc_minmax(false), num_histogram(10), max_levels(INT_MAX),
```

Comment:
As mentioned previously, prefer member initializers; that would let you write `OutputOptions() = default;` here, and it could help make the class trivially default constructible.